### PR TITLE
Clean up dialogue related stuff in PlayState

### PR DESCRIPTION
**Pull request by ggvbo**
Separates dialogue into text files similar to how its being done in the latest weeks

### CODE REVIEW

File: assets/preload/data/tutorial/tutorialDialogue.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+:gf:Hey you're pretty cute.
+:gf:Use the arrow keys to keep up \nwith me singing.
```
Comment (ActualMandM) [inline assets/preload/data/tutorial/tutorialDialogue.txt:2]: no need to have the `\n`

Comment (GNosii) [reply assets/preload/data/tutorial/tutorialDialogue.txt]: removed it

**Comment by hdboye** [comment]
It's never actually used in the game right now, so what's the point of doin it?

**Comment by TentaRJ** [comment]
> It's never actually used in the game right now, so what's the point of changing it?

Future use, I guess? I don't suppose this will be used anytime soon... but it might be good to have.

**Pull request closed without merging**